gradle.properties

```diff
@@ -9,7 +9,7 @@ android.useAndroidX=true
 android.disableAutomaticComponentCreation=true
 
 # Version of the SDK to be built and deployed
-newrelic.agent.version = 7.0.0
+newrelic.agent.version = 7.0.1
```

Comment:
Don't bump the version in a branch, unless it's a `release` branch or explicitly targeted for a release. There won't be a 7.0.1 release, btw.

Reply:
i did these changes to test sonatype for Unity changes.

Reply:
Yea, I figured as much. I also accidentally commit the properties file